src/base/index.vue

```diff
@@ -672,6 +677,25 @@ export default {
           config
         )
         .then((res) => {
+          const sections = res.data.sections;
+          const views = {};
+          sections.map((section) => {
+            this.trackSectionComp(section.name, section.type);
+            if (section.settings) section.settings = JSON.parse(section.settings);
+            if (section.id) {
+              views[section.id] = section;
+            } else {
+              views["test"] = section;
+            }
+          });
+          this.$set(this.displayVariations, this.activeVariation.pageName, {
+            name: this.activeVariation.pageName,
+            views: { ...views },
+          });
+          this.selectedVariation = this.activeVariation.pageName;
+          this.loading = false;
+          this.pageNotFound = false;
+          this.$emit("load", true);
```

Comment:
Please add a comment in the code to explain what does this block of code solve

Reply:
Saw the comment, but what's the point of loading sections for a page that was just created ? it will always be empty @mahfouzhseineweev

Reply:
Indeed, I updated it